### ISSUE TITLE
az upgrade error

### ISSUE THREAD

**Issue by confused-nerd** [Azure/azure-cli#17745]
## Describe the bug

**Command Name**
`az upgrade`

**Errors:**
```
cannot import name 'status_tag_messages'
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 215, in invoke
    args = self.completion.get_completion_args() or args
cli/core/commands/__init__.py, ln 659, in execute
    for exception, id_arg in exceptions:
cli/core/commands/__init__.py, ln 722, in _run_jobs_serially
    exceptions.append((ex, id_arg))
...
lib/python3.6/importlib/__init__.py, ln 166, in reload
    _bootstrap._exec(spec, module)
<frozen importlib._bootstrap>, ln 618, in _exec
<frozen importlib._bootstrap_external>, ln 678, in exec_module
<frozen importlib._bootstrap>, ln 219, in _call_with_frames_removed
azure/cli/core/__init__.py, ln 53, in <module>
    _configure_knack()
azure/cli/core/__init__.py, ln 41, in _configure_knack
    from knack.util import status_tag_messages
ImportError: cannot import name 'status_tag_messages'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az upgrade`

## Environment Summary
```
Linux-5.4.88-12233-g8e0826df26a0-x86_64-with-debian-10.7
Python 3.6.10
Installer: DEB

azure-cli 2.22.0

```

**Comment by yungezz** [comment]
add @fengzhou-msft to have a look.

**Comment by fengzhou-msft** [comment]
@confused-nerd what was the version before you upgrade? This issue was fixed in `2.21.0`, it's likely you were on  a previous version.
The error won't affect other commands in Azure CLI after upgraded, and next time you run `az upgrade`, the error shoun't happen again.

**Comment by confused-nerd** [comment]
Sorry for the delayed response, I have done a subsequent upgrade with no issue as you described.
